pymove/visualization/matplotlib.py

```diff
@@ -0,0 +1,147 @@
+import matplotlib.pyplot as plt
+
+from pymove.utils.constants import DATE, DAY, HOUR, LATITUDE, LONGITUDE, PERIOD, TRAJ_ID
+
+
+def show_object_id_by_date(
+    move_data,
+    create_features=True,
+    kind=None,
+    figsize=(21, 9),
+    return_fig=True,
+    save_fig=True,
+    name='shot_points_by_date.png',
+):
+    """
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=6041015)